xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/ClassEditSheet.xml

```diff
@@ -614,7 +614,9 @@ xcontext.put('propertyCustomDisplayer', new PropertyCustomDisplayer(xcontext))
       #set($queryString = 'wizard=true')
       #if(!$xwiki.exists($homePageRef))
         #set($homePageTitle = '$services.localization.render(''platform.appwithinminutes.appHomePageTitle'', [$doc.space])')
-        #set($queryString = "$queryString&amp;editor=inline&amp;template=AppWithinMinutes.LiveTableTemplate&amp;AppWithinMinutes.LiveTableClass_0_class=$escapetool.url($doc.fullName)&amp;title=$escapetool.url($homePageTitle)&amp;parent=AppWithinMinutes.WebHome")
+        #set($wikiHomePageRef = $services.model.resolveDocument('', 'default'))
+        #set($wikiHomePage = $services.model.serialize($wikiHomePageRef, 'local'))
+        #set($queryString = "$queryString&amp;editor=inline&amp;template=AppWithinMinutes.LiveTableTemplate&amp;AppWithinMinutes.LiveTableClass_0_class=$escapetool.url($doc.fullName)&amp;title=$escapetool.url($homePageTitle)&amp;parent=$wikiHomePage")
```

Comment:
The (default) ``$wikiHomePage`` is going to be URL-safe most of the time but it's still better to URL-encode it to avoid any problems. You can quickly wrap it with ``$escapetool.url`` but a nicer fix would be to call ``$escapetool.url`` only once 2 lines below when the edit URL is created, passing a map that you have created and filled before. This will also help you better format this line (so that it doesn't exceed the 120 chars limit), making it easier to read on GitHub for instance. Ping me if you have questions.